junit-platform-commons/src/main/java/org/junit/platform/commons/support/StringToObjectConverter.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2015-2023 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * https://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junit.platform.commons.support;
```

Comment:
I'm wondering if it wouldn't be better to introduce a `conversion` subpackage here.

@marcphilipp, WDYT?

Reply:
I think it would look tidier with a dedicated subpackage 👍